<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Class</class>
 <widget class="QWidget" name="Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Class</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBoxInherit">
       <property name="text">
        <string>inherit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetInheritance" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboBoxType">
          <item>
           <property name="text">
            <string notr="true">public</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">protected</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">private</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>from</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditBaseClass"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxQObject">
     <property name="text">
      <string>use macro Q_OBJECT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDisableCopy">
     <property name="text">
      <string>declare constructor, move constructor and assignment operator private</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUsePimpl">
     <property name="text">
      <string>use pImpl</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>240</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
